Makefile

```diff
@@ -0,0 +1,29 @@
+PROJECT := ~/go/src/github.com/kubernauts/tk8
+GITCOMMIT := $(shell git rev-parse --short HEAD 2>/dev/null)
+BUILD_FLAGS := -ldflags="-w -X /home/niki/go/src/github.com/kubernauts/tk8/cmd.GITCOMMIT=$(GITCOMMIT)"
```

Comment:
Same here. Can you remove the hard coded paths? It could also be interpolated as `${PROJECT}/cmd.${GITCOMMIT}`

Reply:
@frezbo Sure. It am still working on it.

Reply:
@frezbo Done :)